python/mxnet/gluon/model_zoo/text/base.py

```diff
@@ -94,21 +84,21 @@ def _find_param(block, full_param_name, local_param_name):
 
     return param_dict_results, reg_dict_results
 
-def get_rnn_cell(mode, num_layers, num_embed, num_hidden,
+def get_rnn_cell(mode, num_layers, input_dim, hidden_dim,
```

Comment:
I think in RNNCell it's called hidden_size.

Reply:
input_size/hidden_size then?

Reply:
Yes, it's then consistent with RNNCell.